sym/3695e588-84f7-4928-8a9a-2bb3a4f53363/symbol.lp

```diff
@@ -1,6 +1,6 @@
 (librepcb_symbol 3695e588-84f7-4928-8a9a-2bb3a4f53363
- (name "Generic IC 5-Pin")
- (description "Generic 5-pin integrated circuit.")
+ (name "Generic IC 5-Pin (Variant B)")
```

Comment:
Alternative naming would be "Right Aligned" for this variant and "Left Aligned" for the other variant.

Reply:
...or "(3+2)" resp. "(2+3)". But I'm also fine with "Variant A/B".